### ISSUE TITLE
Support for ARM devices?

### ISSUE THREAD

**Issue by multimill** [Eugeny/terminus#2694]
Hello, 

I am using a Samsung Galaxy Book S with an Snapdragon 8cx. This is a new arm soc and sadly I am not able to run terminus. Do you plan to add support for x86 or ARM devices in the near future?

Thanks in advance.

**Comment by Eugeny** [comment]
There's nothing stopping it from working theoretically, but Terminus, being an Electron app, is performance hungry and probably won't work nicely on ARM.

**Comment by hmsjy2017** [comment]
@Eugeny I do not think so. VS Code can run very well on Raspberry Pi, so Terminus will not have any problems running on ARM devices.
